lib/passport-verify-strategy.ts

```diff
@@ -138,7 +138,8 @@ export class PassportVerifyStrategy extends Strategy {
 /**
  * Creates an instance of [[PassportVerifyStrategy]]
  *
- * NOTE: Should be only used by legacy services using a matching process via MSA
+ * This version of the function should only be used if your service uses the legacy setup. A legacy setup
+ * involves connecting to GOV.UK Verify with a Matching Service Adapter (MSA).
```

Comment:
On second thought here, would it be worth rephrasing to 'involves connecting to GOV.UK Verify with a Matching Service Adapter (MSA) and a service provider', since it's not just the MSA they need to have to connect?

Reply:
Yeah, I've wondered about this too, but I prefer the current wording.  I think the worst likely case is that an RP developer will have to read the comment on the other function too before concluding that this is the one they need.

Reply:
Fair enough. Let's see what users think of the current formulation 👍